jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/format/TraceFormat.java

```diff
@@ -86,7 +86,11 @@
 
 	// constants
 	protected static final String usageMessage = "Usage:\n"
+/*[IF Sidecar18-SE]*/
```

Comment:
Please use the Sidecar18-SE-OpenJ9 flag so all OpenJ9 (Java 8, 9, etc) refer to the traceformat execuable.

Reply:
Fixed, pls review again.